src/main/java/com/parasoft/findings/sonar/MapperUtil.java

```diff
@@ -57,9 +59,16 @@ public static SoftwareQuality mapToSonarImpactSoftwareQuality(String category, b
 
     public static CleanCodeAttribute mapToSonarCleanCodeAttribute(String category) {
         String tag = categoryToTag(category);
-        // TODO will be implemented in CICD-666
-        // The following return value is just for testing, because the default value is CleanCodeAttribute.CONVENTIONAL if not set.
-        return CleanCodeAttribute.CLEAR;
+        for (Map.Entry<String, CleanCodeAttribute> entry : TagPrefixMap.TAG_PREFIX_MAP.entrySet()) {
```

Comment:
Is it better to get the string before first dot in "tag" and check whether the map contains it?